buildSrc/src/main/groovy/org/apereo/gradle/plugin/CasPlugin.groovy

```diff
@@ -13,10 +13,17 @@ import java.security.MessageDigest
  * CAS Gradle plugin implementing user-friendly DSL for building and deploying Apereo CAS server.
  *
  * @author Jonathan Johnson
+ * @author Dmitriy Kopylenko
  * @since 1.0.0
  */
 class CasPlugin implements Plugin<Project> {
 
+    static final CAS_ORIG_SUFFIX = '.casorig'
+
+    static final RESOURCES_DIR = 'src/main/resources'
+
+    static final SRC_DIR = 'src'
```

Comment:
So for example if nothing is changed and we only delete empty `resources` will have empty `src/main` hanging in there. I don't know, I personally have this "OCD" of clean environment as much as possible ;-)